### PR TITLE
Merge fixes

### PR DESCRIPTION
**Pull request by jpaddison3**
All of the ones for this PR only apply to the EA Forum, so this won't apply to LW. The changes are:

* Simple removal of unused files that were causing problems.
* Commenting out of console logs in the old database import that I decided to keep around for some reason.
* Last merge I decided that "Default" was a stupid name for when a post should be "Included". I changed it to "Included". Now it's no longer user-facing, and I need to change it back in a few places.

Altogether a pretty painless merge.

### CODE REVIEW

File: packages/lesswrong/server/database-import/database_import_new.ts [hunk 1]
```diff
@@ -397,119 +379,6 @@ const legacyPostToNewPost = (post, legacyId, user) => {
   };
 }
 
-// / what tags do we get
```
Comment (colophonemes) [inline packages/lesswrong/server/database-import/database_import_new.ts:400]: Why are we deleting all of this? Seems somewhat useful?

File: packages/lesswrong/server/database-import/database_import_new.ts [hunk 2]
```diff
@@ -10,23 +10,23 @@ import mapValues from 'lodash/mapValues';
...
 import * as _ from 'underscore';
 import { Random } from 'meteor/random';
 
 const postgresImportDetails = {
   host: 'localhost',
   port: 5432,
-  database: 'oldforum',
-  user: 'jpaddison', // If this is the logged-in user on localhost, no need for password
-  password: ''
+  database: 'reddit',
```
Comment (colophonemes) [inline packages/lesswrong/server/database-import/database_import_new.ts:19]: What's the story with this change? Is this just moving to more sensible defaults?

File: packages/lesswrong/server/database-import/database_import_new.ts [hunk 3]
```diff
@@ -118,7 +113,7 @@ Vulcan.postgresImport = async () => {
   commentData = _.map(commentData, (comment, id) => addParentCommentId(comment, legacyIdToCommentMap.get(comment.legacyParentId) || commentData[comment.legacyParentId]))
 
   //eslint-disable-next-line no-console
-  console.log("Finished Comment Data Processing") //, commentData[25], commentData[213]);
+  console.log("Finished Comment Data Processing", commentData[25], commentData[213]);
```
Comment (colophonemes) [inline packages/lesswrong/server/database-import/database_import_new.ts:116]: This seems pretty obscure – can you add a comment explaining why these indexes?

**Comment by jpaddison3** [comment]
Basically the reply to your comments is that we're replacing all of our changes with an exact copy of  LW's version of the file. Reasoning being that it's not worth the diff anymore given that it's been 1.5 years since we ran that migration.

**Pull request merged**